specification/security/resource-manager/readme.md

```diff
@@ -78,18 +82,27 @@ These are the global settings for the Security API.
 title: SecurityCenter
 description: API spec for Microsoft.Security (Azure Security Center) resource provider
 openapi-type: arm
-tag: package-composite-v3
+tag: package-preview-2023-06
```

Comment:
```suggestion
tag: package-composite-v3
```
Please don't change the default tag, if the new tag only includes part of the whole RP features.

Reply:
sure